src/components/user/appointments/UserAppointments.js

```diff
@@ -44,12 +58,16 @@ const UserAppointments = () => {
         {appointments.map((item) => (
           <Appointment
             key={item.id}
-            item={item}
+            visitDate={item.visit_date}
+            reason={item.reason}
+            note={item.note}
+            status={item.status}
+            doctor={item.doctor}
```

Comment:
Do you use all of item's fields in Appointent component? You should pass item into it.